package.json

```
@@ -1,8 +1,8 @@
 {
-  "name": "vue-nodelink",
+  "name": "MultiLink",
```

Similar to the comment I made in the MultiMatrix renaming PR, as a package name this should be lowercase (like the GitHub repo name).

```suggestion
  "name": "multilink",
```